ratelimit/ratelimit.go

```diff
@@ -0,0 +1,115 @@
+package ratelimit
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	circularbuffer "github.com/szuecs/rate-limit-buffer"
+)
+
+// Type defines the type of the used breaker: consecutive, rate or
+// disabled.
+type Type int
+
+const (
+	// Header is
+	Header = "X-Rate-Limit"
+	// LocalRatelimitName is the name of the LocalRatelimit, which will be shown in log
```

Comment:
"LocalRatelimitName is the name of the LocalRatelimit, which..." ->
"LocalRatelimitName is the name of the LocalRatelimit filter, which..." ?